tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js

```diff
@@ -216,6 +210,21 @@ describe('PopupTabLayout', function () {
 
 				expect(actual).to.equal(expected);
 			});
+
+			it('should collapse tab only when user enters a menu ', function () {
+				Page.spotlightRight();
+				Page.delay(500);
+				expect(popupTabLayout.isCollapsed).to.be.false();
+
+				Page.spotlightRight();
```

Comment:
I think this step is not needed.

Reply:
As Jenkins performance problems, it needs to some delay.
If remove delay , It occur failure in TV test.

Reply:
Suggested change.